src/js/components/accordion.js

```diff
@@ -63,15 +74,39 @@ module.exports = behavior({
   },
 }, {
   init: root => {
-    forEach(select(BUTTON, root), button => {
+    forEach(root.querySelectorAll(BUTTON), button => {
       const expanded = button.getAttribute(EXPANDED) === 'true';
       toggleButton(button, expanded);
     });
   },
   ACCORDION,
   BUTTON,
-  show: button => toggleButton(button, true),
-  hide: button => toggleButton(button, false),
+  show: showButton,
+  hide: hideButton,
   toggle: toggleButton,
   getButtons: getAccordionButtons,
 });
+
+/**
+ * TODO: for 2.0, remove everything below this comment and export the
+ * behavior directly:
+ *
+ * module.exports = behavior({...});
+ */
+const Accordion = function (root) {
+  this.root = root;
+  accordion.on(this.root);
+};
+
+// copy all of the behavior methods and props to Accordion
+const assign = require('object-assign');
```

Comment:
There's also this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign

But I assume it's not compatible enough with other browsers?